src/communication/azext_communication/manual/_help.py

```diff
@@ -564,25 +564,11 @@
     short-summary: Commands to send emails and get the status of emails previously sent using Azure Communication Services Email service.
 """
 
-helps['communication email status'] = """
-    type: group
-    short-summary: Commands to get the status of emails previously sent using Azure Communication Services Email service.
-"""
-
 helps['communication email send'] = """
     type: command
-    short-summary: "Send an email."
+    short-summary: "Send an email and get final status."
     examples:
```

Comment:
should the example also contain a sample to show how to get the operation status?

Reply:
The Send call also gets the final status